### DISCUSSION TITLE
Can the lock screen clock be reused on the desktop?

### DISCUSSION THREAD

**Discussion by Marain-X** [Mahmud0808/Iconify#553] [General]
I'm really like the lock screen clock which clock style 14 ,I searched many places but couldn't find it name. So I wonder if it has a name or a source.thank you for your time.

**Comment by Mahmud0808** [comment]
Unfortunately these aren't available as widgets. But you can replicate the styles using KWGT app, can be found on playstore.

**Comment by Reepzvan25** [comment]
What happened to the battery icon area? No battery icons to choose.

**Reply by Mahmud0808** [reply]
Must clear data if you are updating from v6.8 to v6.9
